android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -174,6 +178,24 @@ public void crash() {
         throw new RuntimeException("TEST - Sentry Client Crash (only works in release mode)");
     }
 
+    @ReactMethod
+    public void fetchModules(Promise promise) {
+        try {
+            InputStream stream = this.getReactApplicationContext().getResources().getAssets()
```

Comment:
H: Use try with resources and `BufferedInputStream`.
See AndroidOptionsInitializer on Android.